docs/examples/compute/digitalocean/instantiate_api_v1.0.py

```diff
@@ -2,4 +2,4 @@
 from libcloud.compute.providers import get_driver
 
 cls = get_driver(Provider.DIGITAL_OCEAN)
-driver = cls('client id', 'api key', api_version='v1')
```

Comment:
This is a minor thing - I prefer to be explicit and explicitly specify an api version. This way it's immediately obvious which API version is used.